package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "watson-developer-cloud",
-  "version": "0.10.3",
+  "version": "0.10.4",
```

Comment:
Versions are handle by a new release process. Please **don't modify it.**.
See [RELEASE.md](https://github.com/watson-developer-cloud/nodejs-wrapper/blob/master/RELEASE.md)